content/docs/pipelines/upgrade.md

```diff
@@ -145,6 +123,8 @@ To reinstall Kubeflow Pipelines, follow the [command line deployment
 instructions](/docs/gke/deploy/deploy-cli/), but note the following
 change in the procedure:
 
+1. Warning, when you do `kfctl init ${KFAPP} --other-flags`, you should use a different `${KFAPP}` name from your existing `${KFAPP}`. Otherwise, your existing PVs will be deleted during `kfctl apply all -V`.
```

Comment:
Will the underlying PD deletes? As long as the PD still exist, data won't get lost and I dont think this would be a problem.

Reply:
I should change a term, the underlying PD was deleted.